src/MRSortModel.cpp

```diff
@@ -0,0 +1,77 @@
+#include "MRSortModel.h"
+#include "Category.h"
+#include "Criteria.h"
+#include "PerformanceTable.h"
+
+#include <cstdlib>
+#include <map>
+#include <string>
+
+MRSortModel::MRSortModel(Criteria &crits, PerformanceTable &profs, float lbd,
+                         std::string id)
+    : criteria(crits), profiles(profs) {
+  if (!profiles.isOrdered()) {
+    throw std::invalid_argument(
+        "The given performance table cannot be use as a profile");
+  }
```

Comment:
Match the name of the PerformanceTable argument with the header file ie PerformanceTable &profiles.

Reply:
Yeah this was done in purpose, I like that fact that in the header file and thus in the doc we have explicit variable names, however if I keep them that way in the cpp file the implementation is a bit disturbing as they would have the same name as the class attributes: 
`criteria(crits), profiles(profs)` would become `criteria(criteria), profiles(profiles)`

open to suggestion though I was out of idea so I found this turnaround